docs/using-cinder-csi-plugin.md

```diff
@@ -59,6 +59,14 @@ $ csc identity plugin-capabilities --endpoint tcp://127.0.0.1:10000
 CONTROLLER_SERVICE
 ```
 
+#### Get controller implemented capabilities
+```
+$ csc controller get-capabilities  --endpoint tcp://127.0.0.1:10000
```

Comment:
Thanks for improving CSI docs, Are you able to get the output?? as i cannot see ControllerGetCapabilities implemented here 
https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/csi/cinder/controllerserver.go

Reply:
actually, the output is listed here: 

&{type:CREATE_DELETE_VOLUME }
&{type:PUBLISH_UNPUBLISH_VOLUME }

so I believe it's the parent class handled this request and seems no need for us to override this method?

Reply:
I mean , the command does give the output as listed above....

Reply:
IMO, 'output a readable example' should be more helpful, like Line 75.

Reply:
@jichenjc ok, so it think we are getting the default output
@FengyunPan2 IMHO, we should show the actual ouput in doc also, and the actual output has this weird format, so i think it is fine as of now.

Reply:
+1 : show the actual output